### ISSUE TITLE
A given value of an option is erased by the default value 

### ISSUE THREAD

**Issue by chasset** [leo/args#90]
The script `program` defines its option like this:
```
args.option('parameters', "Your parameters", []);
const flags = args.parse(process.argv);
console.log(flags);
```

A test in a terminal returns an empty array:
```
program command --parameters 10 20
```

**Comment by ntwcklng** [comment]
Hey, for every item in your `<list>` you have to call the parameter again. Like this: 
```
$ program command --parameters 10 --parameters 20
or
$ program command -p 10 -p 20
```
BTW: if you want to access the `parameters` flag in your subcommand `command` you have to define it there and not in your main binary `program`.

**Comment by chasset** [comment]
Ok with 2 items or more, but why it doesn't work on a single item?

**Comment by ntwcklng** [comment]
Good catch! I will have a look

**Comment by ntwcklng** [comment]
@chasset fixed in 3.0.0, thanks for your bug report! 🙏

**Comment by leo** [comment]
💃 👍
